package-lock.json

```diff
@@ -4178,14 +4178,12 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
Undo changes in this file*

Reply:
Yeah, done. I don't know how these came in the first place.

Reply:
I've created an issue for the same (#6736), @NishealJ is looking into it.

Reply:
Ok, thanks!